frontend/scss/components/organisms/format-cards.scss

```diff
@@ -15,93 +15,97 @@
 @import 'components/atoms/_text.scss';
 
 .#{utility('format-cards')} {
-  display: flex;
-  flex-direction: column;
-  flex-wrap: wrap;
-  justify-content: space-between;
-  padding: 0;
-
-  @media (min-width: 600px) {
-    flex-direction: row;
-  }
-  @media (min-width: 768px) {
-    flex-direction: column;
-  }
-  @media (min-width: 930px) {
-    padding: 0 60px 75px 60px;
-  }
-  @media (min-width: 1024px) {
-    flex-direction: row;
-  }
+  grid-column: 3 / 23;
 }
 .#{organism('format-cards')} {
-  &-text {
-    width: 100%;
-  }
+  display: grid;
+  grid-column-gap: 30px;
+  grid-template-columns: repeat(12, 1fr);
+  align-items: stretch;
+  margin-top: 20px;
 
   &-card {
-    box-shadow: 6px 9px 40px 0px rgba(0,0,0,0.1);
-    border-radius: 10px;
-    padding: 15px 25px;
-    margin: 20px 0;
     position: relative;
-    transition: 0.3s cubic-bezier(0.25, 0.1, 0.25, 1);
+    grid-column: span 12;
+    margin: 25px 0 0 0;
+    border-radius: 10px;
     background-color: color('white');
-
-    &:hover {
-      box-shadow: 12px 18px 80px 10px rgba(0,0,0,0.05);
-    }
+    box-shadow: 0 15px 30px 0 rgba(0,0,0,.15);
+    transition: transform .3s cubic-bezier(.25,.1,.25,1), box-shadow .3s cubic-bezier(.25,.1,.25,1);
 
     @media (min-width: 600px) {
-      max-width: 31%;
-    }
-    @media (min-width: 768px) {
-      max-width: 100%;
-    }
+      grid-column: span 6;
+  	}
+
     @media (min-width: 1024px) {
-      max-width: 31%;
-    }
+      grid-column: span 3;
+  	}
 
-    &-text {
-      width: 100%;
+    &:hover {
+      transform: translateY(-.125em);
+      box-shadow: 0 25px 60px 0 rgba(0,0,0,.1);
     }
 
+    &-body {
+      height: calc(100% - 47px);
+      min-height: 145px;
+      padding: 25px;
+    }
     &-icon {
       position: absolute;
-      left: -10px;
-      top: -10px;
-      height: 60px;
-      width: 60px;
-      border-radius: 50%;
+      left: 23px;
+      top: 23px;
+      height: 35px;
+      width: 35px;
+    }
+    &-headline {
+      @include txt-font-accent;
+      margin: 6px auto 20px 45px;
+    }
+    &-link {
+      height: 47px;
+      padding: 10px;
+      background-color: color('white');
+      border-radius: 0 0 9px 9px;
+      svg {
+        fill: color('blue-ribbon');
+      }
 
-      &-websites {
-        @include gradient-websites;
-        fill: color('white');
+      &.ap-m-lnk:hover {
+        .ap-m-lnk-icon, .ap-m-lnk-text {
+            transform: none;
```

Comment:
Additionally it might be worthwile to store the parent selector in a variable as I've learnt from Sebil like so:

```scss
.ap-m-lnk {
       $root: &;
        #{$root}-icon, .#{$root}-text {
            transform: none;
```

Reply:
Not sure if it is working here, because of the hover selector, therefore I used .#{molecule('lnk')} on all three